x-pack/filebeat/input/cel/input.go

```diff
@@ -831,6 +833,16 @@ func checkRedirect(cfg *ResourceConfig, log *logp.Logger) func(*http.Request, []
 	}
 }
 
+// retryErrorHandler returns a retryablehttp.ErrorHandler that will log retry resignation
+// but return the last retry attempt's response and a nil error so that the CEL code
```

Comment:
May be the comment should reflect the change made?

Reply:
The current text is correct; the nil error is in response to retry resignation.

Reply:
But is that not returning the last response along with a definite error [ Because we are in ErrorHandler part ]

Otherwise the ErrorHandler will not be called - [the code before the ErrorHandler call](https://github.com/hashicorp/go-retryablehttp/blob/main/client.go#L734-L735).

Reply:
True. That need not be the cause here.. But the statement says `return the last retry attempt's response and a nil error`. But we are returning a non-nil error [ always] if the code reaches this point. Then, we just log the error in this func and return `nil` to the CEL code ?

Reply:
>  But the statement says `return the last retry attempt's response and a nil error`. 

I'm going to push back here. This is not what it says; the context is "*retry resignation* but return the last retry attempt ... and a nil error". It makes no comment on error state returned in the general case.